source/preferences/troubleshoot-notifications.rst

```diff
@@ -0,0 +1,144 @@
+
+  .. tab:: Chrome
+
+    Grant notification permissions for Mattermost in your Chrome web browser.
+
+    1. From the Chrome menu, select **Settings**.
+    2. Select **Privacy and Security** in the left pane.
+    3. Expand **Site settings**. 
+    4. Under **Permissions**, expand **Notifications**.
+    4. Enable the **Sites can ask to send notifications** option, and add your Mattermost workspace URL to the **Allowed to send notifications** list. 
+
+  .. tab:: Edge
+
+    Grant notification permissions for Mattermost in your Edge web browser. 
+
+    1. From the Edge menu, select the **Settings and more (...)** option. 
+    2. Select **Cookies and Site Permissions** in the left pane. 
+    3. Under **All Permissions**, **select Notifications**.
+    4. Add your Mattermost workspace URL to the **Allow** section.
+
+  .. tab:: Firefox
+
+    Grant notification permissions for Mattermost in your Firefox web browser. 
+
+    1. From the Firefox menu, select **Settings**.
+    2. Select **Privacy and Security** in the left pane.
+    3. Under **Permissions**, select the **Settings** option for **Notifications**.
+    4. Enable notifications for your Mattermost workspace URL in the **Permissions** list.
+
+  .. tab:: Safari
+
+    Grant notification permissions for Mattermost in your Safari web browser. 
+
+    1. From the Safari menu, select **Preferences**.
+    2. Select **Websites** and select **Notifications** in the left pane.
+    3. Enable notifications for your Mattermost site.
+
+We recommend checking your operating system settings next.
+
+Check your Operating System settings
+------------------------------------
+
+The operating system you're running Mattermost on can also block Mattermost notifications. Select the **Linux**, **macOS**, or **Windows** tab based on your OS:
+
+.. tab:: Linux
+
+    If you're using Mattermost on a Linux machine, you must enable pop-up notifications from Mattermost and turn off Do Not Disturb mode. If you don't, Linux can block you from receiving Mattermost notifications.
+
+    1. Open **Settings**, then go to **Notifications**.
+    2. Enable the **Show Pop-up Notifications** option.
+    3. Enable the **Show Notification Center** to review and manage your Mattermost notifications. 
+
+    Also ensure that **Do Not Disturb** mode is turned off.
+
+.. tab:: macOS
+
+    If you're using Mattermost on a macOS machine, you must enable application notifications for Mattermost and turn off focus mode. If you don't, macOS can block you from receiving Mattermost notifications.
+    
+    1. Open **System Settings** and go to **Notifications**.
+    2. Under **Application Notifications**, ensure notifications are enabled.
+
+    Also ensure that macOS’ **Focus mode** is turned off.
```

Comment:
We can link macOS docs for turning off Focus mode (https://support.apple.com/en-ca/guide/mac-help/mchl999b7c1a/mac) here.